src/features/amUI/common/DelegationModal/Role/RoleStatusMessages.tsx

```
@@ -37,41 +37,55 @@ export const RoleStatusMessage = ({ role }: RoleStatusMessageProps) => {
-        fontSize='1.5rem'
-        className={classes.inheritedInfoIcon}
-      />
-      <DsParagraph data-size='xs'>
-        <Trans
-          i18nKey={STATUS_TRANSLATION_KEYS[status.type]}
-          values={{
-            user_name: formattedUserName,
-            via_name: formattedViaName,
-          }}
+    return (
+      <div className={classes.infoLine}>
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)